README.md

```diff
@@ -193,6 +215,8 @@ A map [array of objects] of the column names, width, starting point, type of val
 - width (Required) | Length of Fixed Section
 - padding_position (Optional. Default: "start") | Where padding should start ("start" or "end")
 - padding_symbol (Optional. Default: space " ") | What empty space should be padded with (any symbol, letter or number)
+- padding_symbol (Optional. Default: space " ") | What empty space should be padded with (any symbol, letter or number)
```

Comment:
This is in here twice. Sorry for the double comment it was done from the wrong account.